fl4health/strategies/flash.py

```diff
@@ -0,0 +1,164 @@
+from typing import Callable, Dict, List, Optional, Tuple, Union
+
+import numpy as np
+from flwr.common import (
+    FitRes,
+    MetricsAggregationFn,
+    NDArrays,
+    Parameters,
+    Scalar,
+    ndarrays_to_parameters,
+    parameters_to_ndarrays,
+)
+from flwr.server.client_proxy import ClientProxy
+from flwr.server.strategy import FedOpt
+
+
+class FLASH(FedOpt):
```

Comment:
The standard naming convention that we've adopted is that acronyms (like FLASH) when part of classes are camel cased. So FLASH would be Flash or PerFCL would be PerFcl.

Reply:
got it, updated and will do this for new methods

Reply:
Sounds good! There is a bit of ambiguity in the style guides about how to case acronyms. We've just selected this convention.